### PR TITLE
Cleanups

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

File: libethereum/BlockChain.cpp [hunk 1]
```diff
@@ -181,9 +180,6 @@ void BlockChain::init(ChainParams const& _p, std::string const& _path)
 	m_sealEngine.reset(m_params.createSealEngine());
 	m_genesis.clear();
 	genesis();
-
-	// remove the next line real soon. we don't need to be supporting this forever.
```
Comment (chriseth) [inline libethereum/BlockChain.cpp]: Only took two weeks.

File: libwebthree/Support.cpp [hunk 2]
```diff
@@ -111,7 +111,6 @@ void Support::hintSHA256(h256 const& _content, h256 const& _sha256, Secret const
 Address Support::icapRegistrar() const
```
Comment (chriseth) [inline libwebthree/Support.cpp]: Is this used somewhere?

Comment (chfast) [inline libwebthree/Support.cpp]: My IDE says yes, but it might be wrong. Have not check details.

**Comment by chfast** [comment]
I also removed this Support thing.

**Pull request merged**